src/Core/WinSWCore/Configuration/YamlConfiguration.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections.Generic;
+
+namespace winsw.Configuration
+{
+    public class YamlConfiguration
+    {
+        public string? id;
```

Comment:
Isn't it mandatory for the service operation?

Reply:
Yes. Should be changed